pipeline_dp/accumulator.py

```diff
@@ -352,11 +352,15 @@ class SumParams(AccumulatorClassParams):
 class SumAccumulator(Accumulator):
 
     def __init__(self, params: SumParams, values):
-        self._sum = sum(values)
         self._params = params
+        self._sum = 0
+        for value in values:
```

Comment:
Nit: we can do it in 1 line (it's Python 😄 ) 
self._sum += np.clip(values, ...).sum()

and it will be more effective, since numpy vectorization is usually faster than loops